modules/mobile/spec/request/legacy_user_logout_request_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
```

Comment:
I extracted this from the user request specs because the underlying code is specific to IAM and I wanted it to be clear that this is legacy and should be deleted when we finally sunset SIS. 

The SIS equivalent of logout exists in the SIS code, not ours.